Configure

```diff
@@ -561,6 +561,9 @@ $config{build_type} = "release";
 
 my %unsupported_options = ();
 my %deprecated_options = ();
+# If you change this, update apps/version.c
+my @known_seed_sources = qw(getrandom devrandom os egd none);
```

Comment:
For Windows and VMS, it's an error if don't do "os" -- I don't check to see if you do anything else.   I don't think it's worth doing more then that. For unix/linux, "os" implies "devrandom".
These checks are all done at build time using the error directive.  See the various rand_xxx in this PR.